Cogs/verification.py

```diff
@@ -1,11 +1,10 @@
 import discord
 from discord.commands import SlashCommandGroup
 from discord.ext import commands
-from asyncio import TimeoutError as asyncioTimeout
 from pytz import timezone
 
 from Cogs.db import dbconnect
-from Cogs.Helpers.verificationHelpers import insert_verified_user_record, verify_user, get_verification_record
+from Cogs.Helpers.helpers import insert_verified_user_record, verify_user, get_verification_record

```

Comment:
Im gonna be honest, I like leaving it as verification Helpers, since I'm adding a website helper and autocomplete helper file